src/main/java/net/minecraftforge/items/CapabilityItemHandler.java

```diff
@@ -8,15 +8,14 @@
 import net.minecraftforge.common.capabilities.Capability;
 import net.minecraftforge.common.capabilities.CapabilityManager;
 import net.minecraftforge.common.capabilities.CapabilityToken;
+import net.minecraftforge.common.capabilities.ForgeCapabilities;
 import net.minecraftforge.common.capabilities.RegisterCapabilitiesEvent;
 
+@Deprecated(forRemoval = true, since = "1.19.2")
 public class CapabilityItemHandler
```

Comment:
Deprecation javadoc to redirect users to the new class (or making their own token).
```java
/**
 * @deprecated Create your own reference using {@link CapabilityManager#get(CapabilityToken)}, or use {@link ForgeCapabilities#ITEM_HANDLER}.
 */
```